.github/workflows/integration-test.yml

```diff
@@ -106,5 +106,11 @@ jobs:
     permissions:
       actions: write
     steps:
-      - uses: actions/checkout@v4
-      - uses: gautamkrishnar/keepalive-workflow@v2
+      - name: Re-enable workflow
```

Comment:
I think you still need a checkout or hardcode the name of the yml file

Reply:
Could you explain why? I don't see anything that would necessitate it so I'd like to understand what I'm missing.

Reply:
the original action additionally has

```bash
case "${GITHUB_WORKFLOW_REF:?}" in
          "${GITHUB_REPOSITORY:?}"/.github/workflows/*.y*ml@*) ;;
          *) false ;;
        esac
```

which somehow globs for the yml files. Not that I look at it again, I'm not sure it's required. 

In any case `${workflow}` in the API call should be the name of the workflow yml file. But I'm not really sure what the expression `${GITHUB_WORKFLOW_REF%%@*}` evaluates to.